ops/charlie-deploy.tmpl.yaml

```diff
@@ -51,26 +60,28 @@ extraEnvVars: &envVars
     value: "/app/samvera/hyrax-webapp/solr/config"
   - name: DB_ADAPTER
     value: mysql2
+  - name: DEPENDENCIES_NEXT
+    value: "1"
   - name: MYSQL_HOST
-    value: ams-prod-rails-db.cmrjt7rckp5r.us-east-1.rds.amazonaws.com
+    value: ams-demo-rails-db.cmrjt7rckp5r.us-east-1.rds.amazonaws.com
   - name: MYSQL_DATABASE
     value: hyrax
   - name: MYSQL_USER
     value: hyrax
   - name: FCREPO_BASE_PATH
     value: /prod
   - name: FCREPO_HOST
-    value: 10.0.4.85:8080
+    value: 10.0.4.190:8080
   - name: FCREPO_PATH
     value: /fcrepo-webapp-4.7.5/rest
   - name: FCREPO_URL
-    value: http://10.0.4.85:8080/fcrepo-webapp-4.7.5/rest
+    value: http://10.0.4.190:8080/fcrepo-webapp-4.7.5/rest
   - name: IN_DOCKER
     value: "true"
   - name: LD_LIBRARY_PATH
     value: /app/fits/tools/mediainfo/linux
   - name: RAILS_CACHE_STORE_URL
-    value: redis://:prod@ams-charlie-redis-master:6379/ams-prod
+    value: redis://:demo@ams-demo-redis-master:6379/ams-demo
```

Comment:
I think this needs to be `:demo@ams-demo-redis-master.ams-demo.svc.cluster.local` and `ams-demo-redis-master.ams-demo.svc.cluster.local` every where else in the file. 

However it might work as is, so we can try it if you'd like and adjust if it doesn't work

Reply:
it didn't work

Reply:
Hm, but delta did without this change